src/vrep_plugin.cpp

```diff
@@ -782,6 +786,72 @@ static mc_control::MCGlobalController & gc()
   return *gc_ptr;
 }
 
+static void setup_datastore()
+{
+  auto & ctl = gc_ptr->controller();
+  auto add_datastore_entries = [](mc_control::MCController & ctl) {
+    auto & ds = ctl.datastore();
+    /**
+     * @brief Controllers can check whether they are running in CoppeliaSim if this flag is present.
+     */
+    ds.make<bool>("CoppeliaSim", true);
```

Comment:
Maybe we can name this `InterfaceIsCoppeliaSim`?

Reply:
Given that all the other callbacks are nested within the `CoppeliaSim::` prefix, I felt that using the prefix itself was logical. But that's up for debate and we could also make `InterfaceIs***` a convention.

Reply:
> But that's up for debate and we could also make InterfaceIs*** a convention

Yes. This is what I had in mind indeed but your reasoning makes sense as well.